components/Form/subscription.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
 import Button from '../Buttons/button';
 
-function Subcription() {
+function Subscription() {
 	return (
-		<div className='mt-[106px] subscription container flex justify-center'>
-			<div className='w-[1024px] min-h-[253px] lg:py-10 lg:w-full flex flex-col items-center mt-[106px]'>
+		<div className='mt-0 md:mt-[106px] subscription container flex justify-center'>
+			<div className='mt-[106px] lg:mt-0 w-[1024px] min-h-[253px] lg:py-10 lg:w-full flex flex-col items-center'>
 				<h3 className='text-[32px] text-white lg:text-center'>
 					Subscribe for AACoT’24 updates!
 				</h3>
 				<a href='https://www.asyncapi.com/newsletter' target='_blank' rel="noreferrer" className='sm:w-full'>
 				<Button
-					type='submit'
-					className='w-full md:w-[200px] mt-8 px-10' 
-				>
-					Subscribe
-				</Button>
+						type='submit'
+						className='w-full md:w-[200px] mt-8 px-10'
+					>
+						Subscribe
+					</Button>
```

Comment:
@ALIPHATICHYD @ashmit-coder why so much whitespace though? :thinking:

Reply:
I'm so sorry about that, I'd make sure I adjust that 
I didn't know it will come out that way.

Reply:
True, please look into it.

Reply:
> I'm so sorry about that, I'd make sure I adjust that I didn't know it will come out that way.

No problem. It happens sometimes due to auto-formatting extensions running wild. Thanks for the PR @ALIPHATICHYD